.github/settings.yml

```diff
@@ -0,0 +1,26 @@
+# For documentation on settings, see https://github.com/apps/settings
+
+repository:
+  # For options, see https://docs.github.com/en/rest/reference/repos#update-a-repository
+
+  # The name of the repository. Changing this will rename the repository
+  name: hugo-netlify-starter
+
+  # A short description of the repository that will show up on GitHub
+  description: Static website template for CNCF projects
+
+  # A URL with more information about the repository
+  homepage: https://cncf-hugo-starter.netlify.com/
+  
+collaborators:
+  # For options, see https://docs.github.com/en/rest/reference/repos#add-a-repository-collaborator
+  - username: caniszczyk
+    permission: admin
+  - username: celestehorgan
+    permission: admin
+  - username: chalin
+    permission: admin
+  - username: nate-double-u
+    permission: admin
+  - username: sftim
+    permission: admin
```

Comment:
Should @sftim be an admin in this repo? (Do you want to be an admin on this repo @sftim?)

We should add @kapunahelewong to this list as well.

Reply:
I'm happy to be an admin here (hi from Kubernetes SIG Docs)

Reply:
I was just going to edit my comment to say I'd be happy if you were an admin Tim, I was just not aware of any chats that suggested you wanted to be.

Welcome, and thanks :)